src/main/java/org/elasticsearch/rest/BytesRestResponse.java

```diff
@@ -163,4 +171,22 @@ private static void buildThrowable(Throwable t, XContentBuilder builder) throws
             builder.endObject();
         }
     }
-}
\ No newline at end of file
+
+    /*
+     * Builds a simple error response that only will print out the message of the first ElasticsearchException
+     */
+    private static void buildSimpleErrorField(Throwable t, XContentBuilder builder) throws IOException {
+        int counter = 0;
+        while (t != null) {
+            if (counter++ > 10) {
+                break;
```

Comment:
Won't this cause no exception to be added at all? Shouldn't we be returning something in this case still?